src/components/notes/editor-view.jsx

```diff
@@ -97,6 +98,7 @@ function EditorView(props) {
             t,
             setUploading
           )}
+          primaryToolbarComponents={<SharingBannerPlugin />}
```

Comment:
Here we just inject the SharingBanner _inside_ the Atlaskit editor, as a custom component.

Reply:
I have to see if the Editor can handle this props dynamically since Notes will manage the Banner display